src/__tests__/components/social.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import promiseMiddleware from 'redux-promise-middleware';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import { Social } from '../../components/social/social';
+
+const middlewares = [thunk, promiseMiddleware];
+const mockStore = configureMockStore(middlewares);
+const store = mockStore({});
+describe('Render SocialLogin component', () => {
+  it('to have wrapper class', async () => {
+    const wrapper = mount(<Provider store={store}><Social /></Provider>);
+    const btn = wrapper.find('.twitter').first();
+    btn.simulate('click');
+    expect(btn.length).toBe(1);
```

Comment:
I have realised that you not actually testing what the button will do after stimulation. This is universal for All the three buttons, Please kindly indicate those tests there